src/ray/raylet/scheduling/cluster_resource_scheduler_test.cc

```diff
@@ -181,7 +181,7 @@ class ClusterResourceSchedulerTest : public ::testing::Test {
     // The legacy scheduling policy is easier to reason about for testing purposes. See
     // `scheduling_policy_test.cc` for comprehensive testing of the hybrid scheduling
     // policy.
-    RayConfig::instance().initialize("scheduler_hybrid_scheduling,false;");
+    RayConfig::instance().initialize(R"({"scheduler_hybrid_scheduling": false})");
```

Comment:
What is this R for btW?

Reply:
It's the C++ raw string literal. It simplifies character escaping and makes JSON strings more readable. I just found this nice feature in C++ 11. See https://en.cppreference.com/w/cpp/language/string_literal.